server/index.js

```diff
@@ -23,16 +23,19 @@ database.once('connected', () => {
 
 // Model imports
 require('./models/testModel');
+require('./models/submissionModel');
```

Comment:
this format is going to be replaced with the one in my onboarding project solution, reference it to update the imports